CMakeLists.txt

```diff
@@ -177,26 +179,39 @@ if (NOT HIPIFY_CLANG_TESTS_ONLY)
     DESTINATION .
     USE_SOURCE_PERMISSIONS
     PATTERN "hipify-perl"
-    PATTERN "*.sh")
+    PATTERN "*.sh"
+    PATTERN "findcode.sh" EXCLUDE
+    PATTERN "finduncodep.sh" EXCLUDE)
+
+  set(HIPIFY_LIBEXEC_INSTALL_PATH "${CMAKE_INSTALL_PREFIX}/libexec/hipify")
+  install(
+    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bin/
+    DESTINATION libexec/hipify
+    USE_SOURCE_PERMISSIONS
+    PATTERN "findcode.sh"
+    PATTERN "finduncodep.sh"
+    PATTERN "hip*" EXCLUDE)
 
   # Headers are already included in HIP SDK, so skip those if including HIPIFY in HIP SDK.
-  if(NOT HIPIFY_INCLUDE_IN_HIP_SDK)
-    # Install all folders under clang/version/ in CMAKE_INSTALL_PREFIX path.
-    install(
-      DIRECTORY ${LLVM_DIR}/../../clang/${LIB_CLANG_RES}/
-      DESTINATION .
-      COMPONENT clang-resource-headers
-      FILES_MATCHING
-      PATTERN "*.h"
-      PATTERN "*.modulemap"
-      PATTERN "algorithm"
-      PATTERN "complex"
-      PATTERN "new"
-      PATTERN "ppc_wrappers" EXCLUDE
-      PATTERN "omp-tools.h" EXCLUDE
-      PATTERN "omp.h" EXCLUDE
-      PATTERN "ompt.h" EXCLUDE
-      PATTERN "openmp_wrappers" EXCLUDE)
+  if( HIPIFY_INSTALL_CLANG_HEADERS)
+    if(NOT HIPIFY_INCLUDE_IN_HIP_SDK)
+      # Install all folders under clang/version/ in CMAKE_INSTALL_PREFIX path.
+      install(
+        DIRECTORY ${LLVM_DIR}/../../clang/${LIB_CLANG_RES}/include/
+        DESTINATION include/hipify
```

Comment:
> The current path for clang header files on UNIX is ../include/hipify/include. 

https://ontrack-internal.amd.com/browse/SWDEV-435473
Current paths are /opt/rocm-6.1.0-13116/include/mm3dnow.h and /opt/rocm-6.1.0-13116/include/hipify/mm3dnow.h for cmake install and package install respectively. And I had conferred with Nirmal about the installations paths and he'd recommended this structure I've implemented. "header files should be in /opt/rocm-xxx/include/hipify/*.h"

Reply:
It can't be `/opt/rocm-xxx/include/hipify/*.h` due to clang itself, which searches its resource header files based on `-resource-dir=` (or hipify-clang synonym `--clang-resource-directory=`) which should point to the directory parent to `include`. It means that clang searches for that exact `include` folder name with resource header files. And it means that the current (6.0.2) decision with  `/opt/rocm-xxx/include/hipify/*.h` doesn't work at all with hipify-clang installed separately. 

I've already fixed it in #1383, but it is not a final solution.

My point is to get rid of copying anything to `/opt/rocm-xxx/include/...`. We want to use ROCm LLVM `/../lib/llvm/lib/clang/` (Unix) or `/../lib/clang/` (Windows).